Description: Fix bouncing launcher icons in cinnamon (#51)
 tested against cinnamon 3.6.7
Applied-Upstream: commit:8d72aa49c1403bce1b24523a6576e4482a6f2756
Bug: https://github.com/NicoHood/arc-theme/pull/51
Author: PizzaDude <pizzadudedotca@gmail.com>
Last-Update: 2018-01-27
---
 common/cinnamon/sass/_common.scss | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/cinnamon/sass/_common.scss b/common/cinnamon/sass/_common.scss
index 10926ca..42ef925 100644
--- a/common/cinnamon/sass/_common.scss
+++ b/common/cinnamon/sass/_common.scss
@@ -1512,11 +1512,13 @@ StScrollBar {
   }
 
   .panel-bottom & {
+    margin-top: 0;
     margin-bottom: 0;
   }
 
   .panel-top & {
     margin-top: 0;
+    margin-bottom: 0;
     padding-top: 0;
   }
 
-- 
2.14.1

